lib/fiware-pep-steelskin.js

```diff
@@ -200,12 +200,16 @@ function traceRequest(req, res, next) {
     logger.debug('Request for path [%s] from [%s]', req.path, req.get('host'));
     logger.debug('Headers:\n%j\n', req.headers);
 
-    if (req.headers['content-type'] === 'application/json') {
-        logger.debug('Body:\n\n%s\n\n', JSON.stringify(req.body, null, 4));
-    } else if (req.headers['content-type'] === 'application/xml') {
-        logger.debug('Body:\n\n%s\n\n', req.rawBody);
+    if (req.headers['content-type']) {
+        if (req.headers['content-type'].includes('application/json')) {
+            logger.debug('Body:\n\n%s\n\n', JSON.stringify(req.body, null, 4));
+        } else if (req.headers['content-type'].includes('application/xml')) {
+            logger.debug('Body:\n\n%s\n\n', req.rawBody);
+        } else {
+            logger.debug('Unrecognized body type', req.headers['content-type']);
```

Comment:
Some operations in the CB API use text/plain as content-type. In particular this one:

```
PUT /v2/entities/{E}/attrs/{A}/value
```

which corresponds to this action https://github.com/telefonicaid/fiware-pep-steelskin/blob/master/lib/plugins/orionUrls.js#L112

Could be problematic taking into account this logic?

Reply:
I have checked that actually text/plain is being used in our reference postman collection:

https://github.com/telefonicaid/thinking-cities/blob/master/postman/ThinkingCities%20APIs%20Basic%20Ops.postman_collection.json#L770-L771

Reply:
Really there is no problem, this funcion is just to log request.

Reply:
Fixed in 0e64b62